makefiles/pseudomodules.inc.mk

```diff
@@ -102,6 +102,7 @@ PSEUDOMODULES += shell_hooks
 PSEUDOMODULES += slipdev_stdio
 PSEUDOMODULES += sock
 PSEUDOMODULES += sock_async
+PSEUDOMODULES += sock_aux%
```

Comment:
I personally don't like using wildcards for pseudo-modules. It is basically just laziness and it confuses the build system mightily when there is an actual module that matches that wildcard as well. It is unlikely that this will happen for `sock_aux%` and this is not the first offender, so I won't block. However, I would prefer them written out.

Reply:
Fixed. (But in https://github.com/RIOT-OS/RIOT/pull/14703)